packages/rest/src/rest.server.ts

```diff
@@ -281,7 +282,13 @@ export class RestServer extends Context implements Server, HttpServerLike {
     // See https://github.com/strongloop/loopback-next/issues/433
     if (this._httpHandler) return;
 
-    this._httpHandler = new HttpHandler(this);
+    /**
+     * Check if there is custom router in the context
+     */
+    const router = this.getSync(RestBindings.ROUTER, {optional: true});
```

Comment:
Do we need a `try/catch` block for the `getSync` call here?

Reply:
No, as it uses an `optional` flag.